mungegithub/pulls/lgtm_after_commit.go

```diff
@@ -72,6 +72,11 @@ func (LGTMAfterCommitMunger) MungePullRequest(config *config.MungeConfig, pr *gi
 		return
 	}
 
+	// Don't remove the LGTM from merged PRs
+	if pr.Merged != nil && *pr.Merged {
```

Comment:
Hm-- empirically it does happen, but I didn't verify that this would actually fix it.